Filtered_kernel/include/CGAL/internal/Static_filters/Angle_3.h

```diff
@@ -71,16 +71,14 @@ class Angle_3
   {
     CGAL_BRANCH_PROFILER_3("semi-static failures/attempts/calls to   : Angle_3", tmp);
 
-    Get_approx<Point_3> get_approx; // Identity functor for all points
-                                    // but lazy points
```

Comment:
Should not you also remove the definition of `Get_approx` in `./tools.h`?

Reply:
No, as it is still used in `Triangulation_2/3` and `Robust_intersection_traits_3`.